test/e2e_new/e2e_suite_test.go

```diff
@@ -203,7 +210,9 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	kubetestConfig = &conf.KubetestConfig
 	bootstrapClusterProxy = framework.NewClusterProxy("bootstrap", conf.KubeconfigPath, initScheme())
 	e2eConfig = &conf.E2EConfig
-
+	azs := getAvailabilityZones()
```

Comment:
Assume every AWS API call is backed by DynamoDB on the backend, which does guarantee order of results.